@navikt/core/react/src/util/AnimateHeight.tsx

```diff
@@ -85,7 +85,9 @@ const AnimateHeight: React.FC<AnimateHeightProps> = ({
 
   if (typeof initHeight === "number") {
     // Reset negative height to 0
-    initHeight = height < 0 ? 0 : height;
+    if (typeof height !== "string") {
+      initHeight = height < 0 ? 0 : height;
+    }
```

Comment:
`initHeight` blir satt til `height` rett ovenfor, så kanskje man like godt kan bruke `initHeight` i stedet for `height` her, for å slippe å sjekke typen til `height`? (Ikke testet)
```suggestion
    initHeight = initHeight < 0 ? 0 : initHeight;
```

Reply:
Ja, det er noe her 🤔, skulle egentlig ikke gjøre endringer i denne funksjonen, men det dukket opp en typefeil da jeg testet med nyere TS version. Kan se på det mer grundig i morgen.